### ISSUE TITLE
Could you release the model weights of the pre-trained teacher model?

### ISSUE THREAD

**Issue by chouxianyu** [penghui-yang/L2D#1]
@penghui-yang 
Could you release the model weights of the pre-trained teacher model on the multi-label classification dataset? Especially on COCO and VOC.

**Comment by penghui-yang** [comment]
I have already sent you our existing pretrained teachers. Others were deleted due to the limited storage spaces of our computing servers. Nonetheless, you can easily reproduce them by following the instructions provided in our README file.
